dags/veda_data_pipeline/veda_promotion_pipeline.py

```diff
@@ -1,8 +1,10 @@
 import pendulum
 from airflow import DAG
 from airflow.decorators import task
-from airflow.operators.dummy_operator import DummyOperator as EmptyOperator
+from airflow.operators.empty import EmptyOperator
```

Comment:
Nice. I was wondering about this, we can follow suit when updating other pipelines
> [airflow.operators.dummy](https://airflow.apache.org/docs/apache-airflow/2.3.2/_api/airflow/operators/dummy/index.html#module-airflow.operators.dummy)
This module is deprecated. Please use [airflow.operators.empty](https://airflow.apache.org/docs/apache-airflow/2.3.2/_api/airflow/operators/empty/index.html#module-airflow.operators.empty).